src/components/manifold-subscription-details/components/PlanSelector.tsx

```diff
@@ -0,0 +1,22 @@
+import { h, FunctionalComponent } from '@stencil/core';
+import state from '../store';
+
+interface PlanSelectorProps {
+  productId: string;
+}
+
+const PlanSelector: FunctionalComponent<PlanSelectorProps> = ({ productId }) => {
+  state.productId = productId;
+
+  if (state.isLoading) {
+    return 'Loading...';
+  }
+
+  return (
+    <div class="ManifoldSubscriptionCreate__List__Card">
+      <pre>{JSON.stringify(state.plans, null, 2)}</pre>
+    </div>
+  );
+};
+
+export default PlanSelector;
```

Comment:
Why am I not using the existing plan selector? It seemed like with the new stecil store, both the API internals should be structured very differently. Eventually this will replace the old plan selector and we can get rid of all the messy logic that comes along with it.